src/Agent.Plugins/TestFilePublisher/Finder/ITestFileFinder.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Agent.Plugins.TestFilePublisher
+{
+    public interface ITestFileFinder
```

Comment:
>ITestFileFinder [](start = 21, length = 15)

nitpick: resultsfilefinder

Reply:
nit: wanted the plugin to be named TestResultFilePublisher instead of TestFilePublisher and renaming all other functions accordingly